pkg/controller.v1beta2/tensorflow/util.go

```diff
@@ -49,3 +49,18 @@ func ContainChieforMasterSpec(tfJob *tfv1beta2.TFJob) bool {
 	}
 	return false
 }
+
+func CheckTFJobIsNotPending(tfJob *tfv1beta2.TFJob) bool {
+	tfJobStatus := tfJob.Status.DeepCopy()
+	if tfJobStatus.ReplicaStatuses == nil {
+		tfJobStatus.ReplicaStatuses = map[common.ReplicaType]*common.ReplicaStatus{}
+	}
+
+	runningCount := int32(0)
+	for _, status := range tfJobStatus.ReplicaStatuses {
+		runningCount += status.Active + status.Succeeded + status.Failed
```

Comment:
I suggest renaming runningCount  to createdCount.

Reply:
Done